qemu/tests/balloon_check.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, test, params, env):
             self.params["balloon_test_setup_ready"] = True
             self.ori_gmem = self.get_memory_status()
             self.current_gmem = self.ori_gmem
+            self.pre_mem = self.get_vm_mem(self.vm)
+            self.pre_gmem = self.ori_gmem
```

Comment:
It's really confusing that so many kinds of memory variables. Would you mind making it easier to read, maybe add a comment to explain for each one?

Reply:
Update,thanks.

Reply:
By introducing the new variables, I think we need to separate ori_*mem and pre_*mem with different meanings. So the former one means the memory at the very beginning, the latter one means the memory before balloon, right ? If so, I guess you need update L246,  L256, L260 etc.

Reply:
As we talked, I do some update, some self.ori_mem are kept as could not be placed with self.pre_mem.